src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/classPath_pending.jelly

```diff
@@ -0,0 +1,74 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+    <div class="pane-frame">
+        <p id="pendingClasspathEntries-none">
+            No pending classpath entry approvals.
+        </p>
+        <div id="pendingClasspathEntries">
+            Classpath entries pending approval. (Beware of remote URLs, workspace files, or anything else that might change without you noticing.)
+        </div>
+    </div>
+    
+    <script>
+        var mgr = <st:bind value="${it}"/>;
+        function renderPendingClasspathEntries(pendingClasspathEntries) {
+            if (pendingClasspathEntries.length == 0) {
+                $('pendingClasspathEntries-none').show();
+                $('pendingClasspathEntries').childElements().each(function(e){e.remove()});
+                $('pendingClasspathEntries').hide();
+            } else {
+                $('pendingClasspathEntries-none').hide();
+                $('pendingClasspathEntries').childElements().each(function(e){e.remove()});
+                /*
+                   Create a list like:
+                    <p id="pcp-${pcp.hash}">
+                        <button class="approve" onclick="approveClasspathEntry('${pcp.hash}')">Approve</button> /
+                        <button class="deny"    onclick="denyClasspathEntry('${pcp.hash}')">Deny</button> 
```

Comment:
```suggestion
                        <span class="yui-button"><button class="approve" onclick="approveClasspathEntry('${pcp.hash}')">Approve</button></span> /
                        <span class="yui-button"><button class="deny"    onclick="denyClasspathEntry('${pcp.hash}')">Deny</button></span> 
```

Reply:
classPath_pending = only moved, nothing intented to be changed